docs/2.8.0/docs/kubernetes/03-deploy-dependencies/02-create-certificate-issuers.rst

```diff
@@ -1,39 +1,38 @@
 .. Copyright (c) 2023 Digital Asset (Switzerland) GmbH and/or its affiliates. All rights reserved.
 .. SPDX-License-Identifier: Apache-2.0
 
-Create Certificate Issuers
+Create certificate issuers
 ##########################
 
 Objectives
 **********
 
-* Configure a `Smallstep <https://smallstep.com/>`_ ``cert-manager`` compatible certificate issuer.
-* Deploy Smallstep components to a Kubernetes cluster with Helm.
+* Configure a `smallstep <https://smallstep.com/>`_ cert-manager compatible certificate issuer.
```

Comment:
I was also very confused. I did a lot of searching on their website (there seem to be inconsistencies), and I finally settled on lowercase even though it feels uncomfortable. If you feel strongly, I can change. Otherwise, I'll leave it as is for now.